packages/order-utils/src/signature_utils.ts

```diff
@@ -169,12 +174,16 @@ export const signatureUtils = {
         }
 
         const validatorContract = new IValidatorContract(signerAddress, provider);
-        const isValid = await validatorContract.isValidSignature.callAsync(
-            data,
-            signerAddress,
-            validatorSignature.signature,
-        );
-        return isValid;
+        try {
+            const magicValue = await validatorContract.isValidSignature.callAsync(
```

Comment:
PR #2017 fixed a bug where we weren't checking if the validator contract was approved in the `Exchange` contract. We should really just be calling `Exchange.isValidSignature` here to ensure that is being checked with a single RPC call.

Reply:
We should just refactor this to use devutils entirely right? Is that currently checking if the Validator is approved?

Reply:
Let's follow up after #2017 in #2083